src/main/java/org/spongepowered/common/SpongeImplHooks.java

```diff
@@ -176,6 +176,14 @@ public static double getBlockReachDistance(final EntityPlayerMP player) {
         return 5.0d;
     }
 
+    public static double getEntityReachDistanceSq(final EntityPlayerMP player, Entity entity) {
```

Comment:
Can you just javadoc the method for the reasoning why it's a spongeimplhook (aside from the place that it's used) since in a SpongeVanilla workspace, this won't make much sense as to why it exists if not for SpongeForge ;)

Reply:
Oh. Will do, I didn't see any javadocs for other ImplHooks and decided that it's fine :)